Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -186,9 +187,45 @@ extension XcodeModuleProtocol  {
         return buildSettings.map{ "\($0) = \($1);" }.joined(separator: " ")
     }
 
+    func pkgConfigArgs() -> (Xcc: [String], Xld: [String]) {
+        var xcc = [String]()
+        var xld = [String]()
+        for module in recursiveDependencies {
+            guard case let module as CModule = module, let pkgConfigName = module.pkgConfig else {
+                continue
+            }
+
+            do {
+                let pkgConfig = try PkgConfig(name: pkgConfigName)
+                xcc.append(contentsOf: pkgConfig.cFlags)
+                xld.append(contentsOf: pkgConfig.libs)
+            }
+            catch PkgConfigError.CouldNotFindConfigFile {
+                if let providers = module.providers,
+                    provider = SystemPackageProvider.providerForCurrentPlatform(providers: providers) {
+                    print("note: you may be able to install \(pkgConfigName) using your system-packager:\n")
+                    print(provider.installText)
+                }
+            }
+            catch {
+
+            }
+        }
+        return (Xcc: xcc, Xld: xld)
+    }
+
     private func getCommonBuildSettings(_ options: OptionsType) ->[String: String] {
+        let pkgArgs = pkgConfigArgs()
         var buildSettings = ["PRODUCT_NAME": productName]
         buildSettings["PRODUCT_MODULE_NAME"] = c99name
+        buildSettings["OTHER_SWIFT_FLAGS"] = serializeArray(options.Xswiftc+options.Xcc+pkgArgs.Xcc+["-DXcode"])
```

Comment:
Look at this line carefully. Without `+options.Xcc+pkgArgs.Xcc` sometimes xcode can't find headers